controllers/experiments/torqueBalancingJointControl/src/initController.m

```diff
@@ -0,0 +1,67 @@
+function controlParam  = initController(gains,trajectory,DYNAMICS,FORKINEMATICS,CONFIG,STATE)
+%INITCONTROLLER is the initialization function for iCub balancing controllers
```

Comment:
I could be missing something, but looking at the code it seems that `initController` is called by `forwardDynamics` at each simulation step.. perhaps a better name for this function could be `runController` or something similar?